customer-samples/ConsoleApp/Debian.dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy everything else and build
+COPY . ./
+RUN dotnet publish -c Release -f netcoreapp3.1 -o out
+
+# Build runtime image
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1
+WORKDIR /app
+COPY --from=build /app/out .
+
+# Set up Datadog APM
+ENV TRACER_VERSION=1.11.0
```

Comment:
I think if you don't bump this to `1.12.0` now the version bump tool won't be able to catch it. Do we care?

Reply:
Yeah it's probably good to keep this up-to-date with the latest release. Fixed in 064ea569ab3248e5e99c3dd32857b41231eefe8f